src/xtgeo/well/_well_oper.py

```diff
@@ -560,3 +560,23 @@ def _get_bseries_by_distance(depth, inseries, distance):
 
     res = np.array(bseries, dtype=bool)
     return res
+
+
+def create_surf_distance_log(self, surf, name):
```

Comment:
Is `self` a good name for an argument to static function? (but it should be kept as is for consistency now)

Reply:
These modules belongs to the class; just break into more modules to avoid quite big public modules. Tried earlier with other names, but found self easier in the long run (and it gets a color in the IDE)